Doc/whatsnew/3.9.rst

```diff
@@ -199,6 +199,12 @@ Add :func:`curses.get_escdelay`, :func:`curses.set_escdelay`,
 :func:`curses.get_tabsize`, and :func:`curses.set_tabsize` functions.
 (Contributed by Anthony Sottile in :issue:`38312`.)
 
+dis
+---
+
+:class:`dis.Bytecode` objects now supports equality comparisons. (Contributed
```

Comment:
Grammar/typo: it should be 'objects now support' (objects is plural).

Reply:
thanks